corehq/apps/domain/templates/domain/email/support_handoff.html

```diff
@@ -0,0 +1,28 @@
+<p>
+    Hi There! My name is Robin, and I run the support team at Dimagi. I work with {{ contact_name }} and look forward to helping you use CommCare successfully.
```

Comment:
Yeah, it's a little weird.  Honestly, I find the whole automated email thing a little weird, but they wanted a personal touch, I think.  Have you got an alternative suggestion?